### PR TITLE
Show yapf download command if yapf is not downloaded.

### PR DESCRIPTION
**Pull request by rkooo567**
## Why are these changes needed?

Currently, `./ci/travis/format.sh` fails if `yapf` is not downloaded, and it does not properly instruct users what to do. I
- made it print proper commands to install yapf when it is not downloaded
- Refactored code to define expected `yapf` and `flake8` version in the beginning of the file. 

By this PR, if `yapf` is not downloaded, it shows
```
./ci/travis/format.sh
YAPF not installed. Run pip install yapf==0.23.0 Make sure you are using python 3 interpreter
```

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://ray.readthedocs.io/en/latest/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.

### CODE REVIEW

File: ci/travis/format.sh [hunk 1]
```diff
@@ -5,9 +5,18 @@
 # Cause the script to exit if a single command fails
 set -eo pipefail
 
+FLAKE8_VERSION_REQUIRED="3.7.7"
+YAPF_VERSION_REQUIRED="0.23.0"
+
+YAPF_DOWNLOAD_COMMAND_MSG="Run pip install yapf==$YAPF_VERSION_REQUIRED Make sure you are using python 3 interpreter"
```
Comment (edoakes) [inline ci/travis/format.sh]:
```suggestion
YAPF_DOWNLOAD_COMMAND_MSG="pip install yapf==$YAPF_VERSION_REQUIRED"
```

no need for the Python 3 part, we only support python 3 now :)

File: ci/travis/format.sh [hunk 2]
```diff
@@ -5,9 +5,18 @@
 # Cause the script to exit if a single command fails
 set -eo pipefail
 
+FLAKE8_VERSION_REQUIRED="3.7.7"
```
Comment (edoakes) [inline ci/travis/format.sh]: Are we also printing the same if `flake8` isn't installed?

Comment (rkooo567) [reply ci/travis/format.sh]: Not for now. I will also add it.

File: ci/travis/format.sh (hunk 1 above)
Comment (rkooo567) [reply ci/travis/format.sh]: Fixed!

**Review by edoakes — APPROVED** [review]
LGTM. Did you test it locally?

**Comment by rkooo567** [comment]
Yes! I tested
1) pass yapff instead yapf => fail and show msg properly
2) pass flake8d instead of flake8 => fail and show msg properly 
3) pass correct values for both. => succeed to lint

I didn't fully test from the empty env => download yapf & flake8, but I am sure it will work.

**Pull request merged**